plugin/pkg/scheduler/algorithm/scheduler_interface.go

```diff
@@ -34,6 +34,11 @@ type SchedulerExtender interface {
 	// are used to compute the weighted score for an extender. The weighted scores are added to
 	// the scores computed  by Kubernetes scheduler. The total scores are used to do the host selection.
 	Prioritize(pod *v1.Pod, nodes []*v1.Node) (hostPriorities *schedulerapi.HostPriorityList, weight int, err error)
+
+	// Bind informs the extender about the scheduling decision, before the decision is
+	// conveyed to the apiserver. The binding information contains the pod and the
+	// selected node.
+	Bind(pod *v1.Pod, node string) error
```

Comment:
Prefer to following interface: 

```
Bind(pod *v1.Pod, nodeCandidates []*v1.Node) error
```

In extender, the `node` maybe invalid. If only one node, extender can not re-schedule to other node because unknown scheduler's predicates; it may introduce more community between scheduler & extender.

`nodeCandidates` are prepared by scheduler, so extender search the list to find a vaoid  node. But it seems make affinity/ant-affinity hard to consider assumed Pods, let me check the code and back to you today.